en/docs/develop/creating-artifacts/creating-registry-resources.md

```diff
@@ -27,46 +27,31 @@ Select one of the following options.
 
 ### From the existing template
 
-Use the **From existing template** option if you want to generate a registry resource from a template.
+Use the **From existing template** option if you want to generate a resource from a template.
 
-<img src="{{base_path}}/assets/img/integrate/create_artifacts/new_resource/registry-resource-select-existing-template.png" width="800">
+<img src="{{base_path}}/assets/img/integrate/create_artifacts/new_resource/resource-from-existing-template.png" width="800">
 
-Enter a unique name for the **Resource Name**, and **Artifact Name**. Then select a resource template for the **Template Type** field. In this example, a **WSDL File** template is used.
+Select a resource template for the **Template Type** field. Then enter a unique name for the **Resource Name**. In this example, a **XSLT File** template is used.
 
 ### Import from the file system
 
-Use the **Import from file system** option to import a file or a folder containing registry resources.
+## Step 1: Import a file
```

Comment:
The H2 titles (`##`) used as `Steps` are not positioned correctly. Please review and update. (lines 5,38, and 46)